config/environments/development.rb

```diff
@@ -54,6 +54,9 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
+
```

Comment:
This is actually already in the config, a few entries above. (Its how I realised that the logging was already in Rails 5.2 and grabbed it for SS)

Reply:
Where? I can't see it. I have also done a search through the whole project.

Reply:
lines 47 and 58. One copy of it actually disappeared in your code-review changes, and then it appeared again following you merges, so maybe some merge issues.

Reply:
Wait, ignore me. It has gone

Reply:
I'm assuming this is showing the code as it was when I first made the comment.